### DISCUSSION TITLE
Need to know ECCN for google/gson and CCATS if its ECCN 5D002 or 5D992

### DISCUSSION THREAD

**Discussion by hemantmakhijani** [google/gson#2726] [Q&A] [answered]
Hello, 

We are looking after knowing the ECCN for google/gson

If its 5D002 (or may be 5D992) it should have got CCATS certificate from U.S. as export control license.

**Comment by Marcono1234** [comment]
For future reference, there is an existing issue, https://github.com/google/gson/issues/558 (which you commented on as well).

I don't know the ECCN / cannot answer this question. @eamonnmcmanus, do you have an answer, or know who could answer this?

**Comment by eamonnmcmanus** [comment, marked as answer]
I replied on issue #558.

**Reply by hemantmakhijani** [reply]
Thanks. I think we can close this thread and continue at [#558](https://github.com/google/gson/issues/558).
